project/app/views.py

```diff
@@ -90,7 +60,7 @@ def put(self, request):
         profile_serializer = UserProfileSerializer(instance=user_profile, data=request.data)
```

Comment:
Line: 50:  user = UserProfile.objects.all() for multiple users it should be plural.
and are we not using an Admin permission decorator there in `ProfileView`?

Reply:
Profile view is also used by everyone to view their own profile